docs/source/flags.rst

```diff
@@ -3,142 +3,161 @@
 TornadoVM Flags
 ===============
 
-There is a number of runtime flags and compiler flags to enable experimental features, as well as fine and coarse grain profiling in the context of TornadoVM.
+TornadoVM provides runtime and compiler flags to enable experimental features, tuning, and profiling. These flags fall into two categories:
 
-**Note:** for the following examples ``s0`` represents an arbitrary task-graph, as well as ``t0`` represents a given task's name.
+1. **JVM Flags** (passed with the `-D` prefix via the `--jvm` option)
+2. **TornadoVM CLI Flags** (passed directly to the `tornado` Python wrapper)
 
-All flags needs Java prefix of ``-D``. An example of tornado using a flag is the following:
+.. note::
+   In the examples below, ``s0`` refers to a task graph and ``t0`` to a specific task within that graph.
```

Comment:
I think we can remove this

Reply:
There are some flags that use `s0` and `t0`. So, I thought it would be nice to explain first, what they are.